ui/components/src/model/units/bytes.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { UnitGroupConfig, UnitConfig } from './types';
+
+const bytesUnitKinds = ['Bytes'] as const;
+type BytesUnitKind = typeof bytesUnitKinds[number];
+export type BytesUnitOptions = {
+  kind: BytesUnitKind;
+  decimal_places?: number;
+};
+export const BYTES_GROUP_CONFIG: UnitGroupConfig = {
```

Comment:
Group and unit config is new and helps us populate the various selects in a user-friendly way.